src/components/HorizontalCard.tsx

```diff
@@ -39,14 +19,16 @@ const HorizontalCard: React.FC<IProps> = ({
   className,
   emojiSize,
 }) => (
-  <StyledCard className={className}>
-    <Emoji size={emojiSize} text={emoji} />
-    <Content>
-      <Title>{title}</Title>
-      <Description>{description}</Description>
+  <Flex borderRadius="base" className={className}>
+    <Emoji fontSize={`${emojiSize}rem`} text={emoji} />
+    <Flex grow="0" shrink="1" basis="75%" ml="8">
```

Comment:
@ameeetgaikwad and @pettinarip: Your culprit is this `Flex` component, when it should just be a `Box`. You will need to reword the other props.

```tsx
<Box flexGrow='0' flexShrink='1' flexBasis='75%' ml="8">
```

Reply:
I'll work on that

Reply:
Thanks, @ameeetgaikwad and @TylerAPfledderer. Will circle back in a couple of days

Reply:
@corwintines check this out! I've done the required changes